client/src/components/listViewHelpers.js

```diff
@@ -71,6 +71,11 @@ export function MetadataColumns({item, href, includeNamespace, resourceClass}) {
                     className={resourceClass}
                 />
                 <div className='td_iconLabel'>{item.kind}</div>
+                
+                {/** If the node is a master, display a simple "MASTER" label below the item.kind icon */}
+                {Object.entries(item.metadata.labels).some(([key, _]) => key === "node-role.kubernetes.io/master")
```

Comment:
This should probably read:
`{item.metadata.labels && Object.entries(item.metadata.labels).some(([key, _]) => key === "node-role.kubernetes.io/master")`

Note every item will have labels.

Reply:
Yes sorry for that ;) I'll fix that right away :see_no_evil: